### PR TITLE
Add v7 of sc samples

### PR DESCRIPTION
**Pull request by SzymonPobiega**
Connects to https://github.com/Particular/DotNetCoreLaunch/issues/135

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@Particular/servicecontrol-maintainers @SeanFeldman do you have capacity to review this?

**Comment by SeanFeldman** [comment]
@SzymonPobiega will try to get to it tomorrow. Could you please rebase to resolve conflicts? Thanks.

**Comment by SzymonPobiega** [comment]
@SeanFeldman squashed, rebased and force-pushed the new version.

File: samples/servicecontrol/adapter-sqs/sample.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Prerequisistes
+
+ 1. [Install ServiceControl](/servicecontrol/installation.md). 
+ 2. Using [ServiceControl Management](/servicecontrol/license.md#servicecontrol-management-app) tool, set up ServiceControl to monitor endpoints using MSMQ transport:
+	 
+   * Add a new ServiceControl instance:
+   * Use default `Particular.ServiceControl` as the instance name (ensure there is no other instance of SC running with the same name).
+
+NOTE: If other ServiceControl instances have been running on this machine, it's necessary to specify a non-default port number for API. [Adjust ServicePulse settings](/servicepulse/host-config.md#changing-the-servicecontrol-url) accordingly to point to this location.
```
Comment (mikeminutillo) [inline samples/servicecontrol/adapter-sqs/sample.md]: It's also likely that you won't be able to use the default instance name.

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_2/Adapter/Program.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        var transportAdapter =
+            new TransportAdapterConfig<RabbitMQTransport, RabbitMQTransport>("ServiceControl.RabbitMQ.Adapter");
+
+        transportAdapter.CustomizeServiceControlTransport(
+            customization: transport =>
+            {
+                transport.ConnectionString("host=localhost");
+                var delayedDelivery = transport.DelayedDelivery();
+                delayedDelivery.DisableTimeoutManager();
```
Comment (SeanFeldman) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_2/Adapter/Program.cs]: Needs to be removed based on the [TC error](https://builds.particular.net/viewLog.html?buildId=272957&buildTypeId=DocsParticularNet_Snippets_1Build&tab=buildLog&_focus=355#_state=121).

File: samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_2/Adapter/Program.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+#pragma warning disable 618
+
+        #region EndpointSideConfig
+
+        transportAdapter.CustomizeEndpointTransport(
+            customization: transport =>
+            {
+                transport.ConnectionString("host=localhost");
+                var delayedDelivery = transport.DelayedDelivery();
+                delayedDelivery.DisableTimeoutManager();
```
Comment (SeanFeldman) [inline samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_2/Adapter/Program.cs]: Same as above.

**Comment by SeanFeldman** [comment]
Sme Nuget conflicts (https://builds.particular.net/viewLog.html?buildId=272957&buildTypeId=DocsParticularNet_Snippets_1Build&tab=buildLog&_focus=345#_state=121).

**Comment by SzymonPobiega** [comment]
@mikeminutillo @SeanFeldman done.

**Pull request merged**